fonts.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+	font-family: 'PT Serif';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/second_final_np/pullRequest?prid=6218911)